contrib/arrow-cmake/CMakeLists.txt

```diff
@@ -70,6 +70,10 @@ add_custom_command(OUTPUT orc_proto.pb.h orc_proto.pb.cc
         --cpp_out="${CMAKE_CURRENT_BINARY_DIR}"
         "${PROTO_DIR}/orc_proto.proto")
 
+if (CMAKE_CXX_COMPILER_ID STREQUAL "AppleClang")
```

Comment:
Why do we need this?
(you may add a comment)

Reply:
@alexey-milovidov arrow-cmake cmake file calling orc cmake subroutine which detects certain compiler features. Apple Clang compiler failed to compile this code without specifying c++11 subset. As result these compiler features detected as absent. In result it results in fail of compile orc itself.  In orc makefile there is code that sets flags, but arrow-cmake ignores these flags.

Reply:
Ok. I've copy-pasted your explanation into code.

Reply:
Oh.. you mean to add comment to code?

Reply:
Yes, otherwise comments will be lost :)